src/app/Http/Controllers/CrudFeatures/Reorder.php

```diff
@@ -15,9 +15,8 @@ public function reorder()
     {
         $this->crud->hasAccessOrFail('reorder');
 
-        if (! $this->crud->isReorderEnabled()) {
-            abort(403, 'Reorder is disabled.');
-        }
+        // check if Reorder is disabled
+        if (! $this->crud->isReorderEnabled()) abort(403, 'Reorder is disabled.');
```

Comment:
Comment is redundant.

Please, **always** use blocks after `if` even for just a single statement.

Also, should that be translatable?